tool/release-api-docs.sh

```diff
@@ -32,6 +32,9 @@ mv ../doc/* .
 git reset origin/gh-pages
 git add .
 
+echo "see https://github.com/ajaxorg/ace/issues/5375" > .nojekyll
+git add .nojekyll -f
```

Comment:
Do we need to create this .nojekyll file (which then completely bypasses Jekyll processing) in this script or can we just create it right now and commit it to the master branch?

Reply:
We can create the file outside of it too, i've used the script to not have too many files in repository root, but maybe that would be better than having more complex  script. Do you think we should just commit it to the master?

Reply:
done